static/katex.less

```diff
@@ -442,10 +443,10 @@
     }
 
     .delimsizing {
-        &.size1 { font-family: KaTeX_Size1; }
```

Comment:
For consistency with what? :) If there is literally not fixing anything, we can remove it – but I assume you added it to fix some particular problem you saw.